content/_layouts/post.html.haml

```diff
@@ -42,6 +41,3 @@ layout: default
               = display_author_for(page)
 
         = content
-
-      %div#sidebar.col-md-4.events.vertical
-        = partial('sidebar.html.haml')
```

Comment:
@i386 Ah ok, I thought this change would be forever 😄 

My proposal is:

1. Remove the sidebar for small devides
1. Remove the sidebar for screen resolutions `<`  960px (width)